lib/file-list.js

```diff
@@ -71,14 +71,14 @@ var List = function (patterns, excludes, emitter, preprocess, batchInterval) {
   var self = this
 
   // Emit the `file_list_modified` event.
-  // This function is throttled to the value of `batchInterval`
+  // This function is debounced to the value of `batchInterval`
   // to avoid spamming the listener.
   function emit () {
     self._emitter.emit('file_list_modified', self.files)
   }
-  var throttledEmit = _.throttle(emit, self._batchInterval, {leading: false})
+  var debouncedEmit = _.debounce(emit, self._batchInterval, {leading: false})
```

Comment:
I don't think `{leading: false}` needs to be passed into the function here - debouncing never fires the leading call.